diff --git a/src/app/helpers/updater.ts b/src/app/helpers/updater.ts
deleted file mode 100644
index a7cbacea..00000000
--- a/src/app/helpers/updater.ts
+++ /dev/null
@@ -1,49 +0,0 @@
-import { app, ipcMain } from 'electron';
-import { autoUpdater } from 'electron-updater';
-import * as os from "os";
-
-export default class AppUpdater {
-  constructor(mainWindow) {
-    const log = require("electron-log")
-    log.transports.file.level = "info"
-    const platform = os.platform() + '_' + os.arch();
-    const version = app.getVersion()
-
-    autoUpdater.logger = log;
-
-    autoUpdater.on("update-available", (event) => {
-      log.info("A new update is available")
-    });
-
-    autoUpdater.on("update-downloaded", (updateInfo) => {
-      log.info("update downloaded: "+updateInfo.releaseName);
-      mainWindow.webContents.send("updater", "update-downloaded", updateInfo);
-    });
-
-    autoUpdater.on("error", (error) => {
-      log.error(error)
-    });
-
-    autoUpdater.on("download-progress", (progress) => {
-      log.info("download progress: "+progress.bytesPerSecond+" "+progress.percent);
-    });
-
-
-    autoUpdater.on("checking-for-update", (event) => {
-      log.info("checking-for-update")
-    });
-
-    autoUpdater.on("update-not-available", () => {
-      log.info("update-not-available")
-    });
-
-    ipcMain.on('updater', (event, arg) => {
-        if(arg == "quitAndInstall")
-          autoUpdater.quitAndInstall();
-    });
-    
-    mainWindow.webContents.once("did-frame-finish-load", (event) => {
-      autoUpdater.checkForUpdates();
-    });
-  }
-}
diff --git a/src/electron/index.ts b/src/electron/index.ts
index 32071e78..75891c82 100644
--- a/src/electron/index.ts
+++ b/src/electron/index.ts
@@ -5,7 +5,6 @@ import './dev-extensions';
 
 import { devMenuTemplate } from '../app/menu/dev_menu_template';
 import { editMenuTemplate } from '../app/menu/edit_menu_template';
-import AppUpdater from '../app/helpers/updater';
 
 const windowStateKeeper = require('electron-window-state');
 
@@ -60,10 +59,6 @@ function createWindow() {
     win.webContents.openDevTools();
   }
 
-  if (!DEV_SERVER) {
-    let autoUpdater = new AppUpdater(win);
-  }
-
   // Emitted when the window is closed.
   win.on('closed', () => {
     // Dereference the window object, usually you would store windows
